core/js/nests.maps.js.php

```diff
@@ -0,0 +1,135 @@
+<?php
+	# Send Javascript header
+	header('Content-type: text/javascript');
+
+	# Load variables and locales
+	include_once('../../config.php');
+	$variables = SYS_PATH.'/core/json/variables.json';
+	$config = json_decode(file_get_contents($variables));
+	include_once('../process/locales.loader.php');
+
+	# Load nests-file
+	$nest_file = file_get_contents('../json/nests.stats.json');
```

Comment:
on my setup, which has limited htaccess rights it didn't work without the SYS_PATH